Week3/homework/TipCalculator/app.js

```diff
@@ -0,0 +1,21 @@
+const bill = document.getElementById('bill');
+const service = document.getElementById('service');
+const people = document.getElementById('people');
+const btn = document.getElementById('calc');
+const tip = document.getElementById('tip');
+const multi = document.getElementById('multi');
```

Comment:
It's a good practice to use the variable names in order to distinguish the variables between them.
For example I try to name html elements using the `El` suffix. eg billEl or tipEl
This will help me later to use `tip` in order to name the actual tip variable and not `theTip`. And also I would know that tipEl refers to the html element and can be used only to get or set things to html